### ISSUE TITLE
Published npm package for 0.2.3 throws "just_group_by_1.default is not a function"

### ISSUE THREAD

**Issue by cauli** [staltz/react-native-emoji-modal#3]
### Overview
Tried for the first time to use the library on a Expo project. It failed during runtime with:

```
 ERROR  TypeError: (0, just_group_by_1.default) is not a function. (In '(0, just_group_by_1.default)(this.filteredEmojis, function (emoji) {
          return emoji.category;
        })', '(0, just_group_by_1.default)' is undefined)
```

#### Workaround
Downgrading to `0.2.2` worked for me. 

I diffed both transpiled `index.js`

0.2.2
```javascript
const groupBy = require('just-group-by/index.js');
const mapValues = require('just-map-values/index.js');
```

0.2.3
```javascript
const just_group_by_1 = require("just-group-by");
const just_map_values_1 = require("just-map-values");
```

The culprit seems to be [this commit](https://github.com/staltz/react-native-emoji-modal/commit/6e1e30f8b47a1812d82ef7af1ea986bedebaa8b8)

---

Thank you for the really nice project 🍰

**Comment by staltz** [comment]
Sigh, whatever your bundler is (or what Expo is using) is interpreting `require`s as ESM, which is totally wrong, but I'll issue a dirty fix to this soon.

**Comment by staltz** [comment]
Released 0.2.4
